src/mame/amstrad/amstrad_m.cpp

```diff
@@ -2174,6 +2180,18 @@ The exception is the case where none of b7-b0 are reset (i.e. port &FBFF), which
 	}
 }
 
+uint8_t amstrad_state::amstrad_cpc_mem_r(offs_t offset)
+{
+	if (!machine().side_effects_disabled())
+		m_maincpu->adjust_icount(-((4 - m_maincpu->total_cycles() % 4) % 4));
+	return ((u8*)m_banks[(offset >> 13)]->base())[offset & 0x1fff];
+}
+
+void amstrad_state::amstrad_cpc_mem_w(offs_t offset, uint8_t data)
+{
+	m_maincpu->adjust_icount(-((4 - m_maincpu->total_cycles() % 4) % 4));
+	((u8*)m_banks[(offset >> 13) + 8]->base())[offset & 0x1fff] = data;
+}
```

Comment:
The rest of this file uses unabbreviated `uint8_t` – better to be consistent with the current code.